SRW-SD failed to run generate_FV3LAM_wflow.py on Orion

SRW-RD failed to run ./generate_FV3LAM_wflow.py. Please see the error message below

 ./generate_FV3LAM_wflow.py

  ========================================================================
  Starting experiment generation...
  ========================================================================

        ========================================================================
        Starting function setup() in "setup.py"...
        ========================================================================
0 schema-validation errors found in config

*********************************************************************
FATAL ERROR:
Experiment generation failed. See the error message(s) printed below.
For more detailed information, check the log file from the workflow
generation script: /work2/noaa/naqfc/jhuang/rrfs-sd/ush/log.generate_FV3LAM_wflow
*********************************************************************

Traceback (most recent call last):
  File "/work2/noaa/naqfc/jhuang/rrfs-sd/ush/./generate_FV3LAM_wflow.py", line 843, in <module>
    expt_dir = generate_FV3LAM_wflow(
               ^^^^^^^^^^^^^^^^^^^^^^
  File "/work2/noaa/naqfc/jhuang/rrfs-sd/ush/./generate_FV3LAM_wflow.py", line 75, in generate_FV3LAM_wflow
    expt_config = setup(ushdir, user_config_fn=config, debug=debug)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/work2/noaa/naqfc/jhuang/rrfs-sd/ush/setup.py", line 278, in setup
    expt_config = load_config_for_setup(ushdir, default_config_fp, user_config_fp)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/work2/noaa/naqfc/jhuang/rrfs-sd/ush/setup.py", line 150, in load_config_for_setup
    default_config.dereference(
  File "/work2/noaa/naqfc/jhuang/rrfs-sd/conda/envs/srw_app/lib/python3.12/site-packages/uwtools/config/formats/base.py", line 236, in dereference
    new = jinja2.dereference(val=self.data, context=context or self.data)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/work2/noaa/naqfc/jhuang/rrfs-sd/conda/envs/srw_app/lib/python3.12/site-packages/uwtools/config/jinja2.py", line 142, in dereference
    kd, vd = [dereference(x, context, val, [*keys, k]) for x in (k, v)]
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/work2/noaa/naqfc/jhuang/rrfs-sd/conda/envs/srw_app/lib/python3.12/site-packages/uwtools/config/jinja2.py", line 142, in dereference
    kd, vd = [dereference(x, context, val, [*keys, k]) for x in (k, v)]
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/work2/noaa/naqfc/jhuang/rrfs-sd/conda/envs/srw_app/lib/python3.12/site-packages/uwtools/config/jinja2.py", line 142, in dereference
    kd, vd = [dereference(x, context, val, [*keys, k]) for x in (k, v)]
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/work2/noaa/naqfc/jhuang/rrfs-sd/conda/envs/srw_app/lib/python3.12/site-packages/uwtools/config/jinja2.py", line 148, in dereference
    rendered = _deref_render(val, context, local)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/work2/noaa/naqfc/jhuang/rrfs-sd/conda/envs/srw_app/lib/python3.12/site-packages/uwtools/config/jinja2.py", line 279, in _deref_render
    template = env.from_string(val)
               ^^^^^^^^^^^^^^^^^^^^
  File "/work2/noaa/naqfc/jhuang/rrfs-sd/conda/envs/srw_app/lib/python3.12/site-packages/jinja2/environment.py", line 1111, in from_string
    return cls.from_code(self, self.compile(source), gs, None)
                               ^^^^^^^^^^^^^^^^^^^^
  File "/work2/noaa/naqfc/jhuang/rrfs-sd/conda/envs/srw_app/lib/python3.12/site-packages/jinja2/environment.py", line 771, in compile
    self.handle_exception(source=source_hint)
  File "/work2/noaa/naqfc/jhuang/rrfs-sd/conda/envs/srw_app/lib/python3.12/site-packages/jinja2/environment.py", line 942, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "<unknown>", line 1, in template
jinja2.exceptions.TemplateAssertionError: No filter named 'include'.

The package with ush can be found  at /work2/noaa/naqfc/jhuang/rrfs-sd/ush

--- a/ush/config.smoke_dust.yaml
+++ b/ush/config.smoke_dust.yaml
@@ -21,16 +21,15 @@ workflow:
   DO_REAL_TIME: false
   COLDSTART: true
 #  WARMSTART_CYCLE_DIR: '/path/to/warm/start/files'
+  taskgroups:
+    - parm/wflow/prep.yaml
+    - parm/wflow/smoke_dust.yaml
+    - parm/wflow/coldstart.yaml
+    - parm/wflow/post.yaml
 nco:
   envir_default: test_smoke_dust
   NET_default: smoke_dust
   RUN_default: smoke_dust
-rocoto:
-  tasks:
-    taskgroups: '{{ ["parm/wflow/prep.yaml", "parm/wflow/smoke_dust.yaml", "parm/wflow/coldstart.yaml", "parm/wflow/post.yaml"]|include }}'
-    metatask_run_ensemble:
-      task_run_fcst_mem#mem#:
-        walltime: 02:00:00
 task_get_extrn_ics:
   envvars:
     EXTRN_MDL_NAME_ICS: RAP

